node/core/dispute-coordinator/src/initialized.rs

```diff
@@ -593,9 +574,6 @@ impl Initialized {
 				}
 			},
 			DisputeCoordinatorMessage::RecentDisputes(tx) => {
-				// Return error if session information is missing.
-				self.ensure_available_session_info()?;
```

Comment:
I'm not sure what's the purpose of these checks. I might have to keep them. @sandreim ?

Reply:
I see. Let's keep this for now.